.github/issue_template.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Hi!  Thanks for taking the time to report feedback.
+
+Are you concerned with running your own instance of Cromwell,e.g. config issues?  Then please report your issue to the WDL Forum:
+https://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team
+
+Are you having an issue w/ Firecloud?  Then please report your issue to the Firecloud Forum:
+https://gatkforums.broadinstitute.org/firecloud/categories/ask-the-firecloud-team
+
+
+Are you seeing something that looks like a bug?  Then great!  You're in the right place.
+-->
+
+
+<!-- Which backend are you running?  -->
+
+<!-- Paste/Attach your workflow if possible: -->
+
+<!-- Paste your configuration if possible: -->
```

Comment:
perhaps with a friendly reminder to redact any sensitive information

Reply:
Yeah actually this is super important. There's a lot of sensitive info in there and we 100% don't want people's mysql passwords or google tokens ending up in github...